webcompat/static/js/lib/models/issue.js

```diff
@@ -231,6 +231,12 @@ issueList.IssueCollection = Backbone.Collection.extend({
       delete params.direction;
     }
 
+    // Issue 999, support domain name search
+    // Replace all site:DOMAIN_NAME to DOMAIN_NAME in:title
+    if (!!params.q.match(/site:([\w-\.]+(:\d+)?)/i)) {
```

Comment:
Any reason to use `!!` to coerce to boolean?

`String.prototype.match` will return an array (truthy) if it finds a match, and null if it doesn't (falsey).

I could be missing something though!